contrib/fuzz/unstable_call_graph.py

```diff
@@ -0,0 +1,227 @@
+        return symbol
+
+
+def get_architecture(file_path):
+    try:
+        result = subprocess.run(['file', file_path], capture_output=True, text=True)
+        output = result.stdout.strip()
+
+        if 'x86-64' in output:
+            return 'x86_64'
+        elif 'aarch64' in output or 'arm64' in output:
+            return 'arm64'
+        else:
+            return 'Unknown'
+
+    except FileNotFoundError:
+        print("The 'file' command is not available. Ensure it's installed.")
+        return None
+    except Exception as e:
+        print(f"An error occurred: {e}")
+        return None
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description='''Creates a function hierarchy and highlights unstable functions''',
+    )
+    parser.add_argument(
+        "--afl-edge-id-file",
+        help="Path to afl++'s file containing all edge ids and symbols of the binary",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--afl-fuzzer-stats",
+        help="Path to afl++'s fuzzer_stats file",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--binary-file",
+        help="Path to binary file from which to generate the call graph",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--target-function",
+        help="The target function that is the top of the call graph (e.g. fuzz_target_name_fuzz_target)",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--ignored-prefixes",
+        help="Path to file that contains strings that if at the start of a function name, that function is skipped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--dump-hierarchy",
+        help="Path to json file where the hierarchy will be dumped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--dump-unstable",
+        help="Path to file where unstable symbols will be dumped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--no-tui",
+        help="Whether to output the hierarchy to the terminal",
+        action="store_true")
+
+    args = parser.parse_args()
+
+    unstable_edges = []
+    unstable_functions = set()
+    prefixes_to_ignore = set()
+
+    with open(args.afl_fuzzer_stats, 'r') as fuzzer_stats_file:
+        for line in fuzzer_stats_file:
+            if line.startswith('var_bytes'):
+                unstable_edges = line.split(":")[1].strip().split(" ")
+                break
+
+    with open(args.afl_edge_id_file, 'r') as edge_id_file:
+        for line in edge_id_file:
+            for edge in unstable_edges:
+                if f'edgeID={edge}' in line:
+                    function = line.strip().split(" ")[1].split("=")[1]
+                    if function not in unstable_functions:
+                        unstable_functions.add(function)
```

Comment:
`unstable_functions` is a set, so it'll dedup automatically

```suggestion
                    unstable_functions.add(function)
```